dll/3rdparty/libtirpc/CMakeLists.txt

```diff
@@ -85,10 +84,12 @@ list(APPEND SOURCE
     src/xdr_rec.c
     src/xdr_reference.c
     src/xdr_sizeof.c
-    src/xdr_stdio.c)
+    src/xdr_stdio.c
+    precomp.h)
 
 add_library(libtirpc SHARED
     ${SOURCE}
+    src/rtime.c
```

Comment:
What's special about this file?
(a comment would help future readers)

Reply:
This change is indeed weird, can you explain it at least in the file and perhaps here? Something we need to fix?

Reply:
We get:
```
e:/reactos/dll/3rdparty/libtirpc/src/rtime.c: In function 'rtime':
e:/reactos/dll/3rdparty/libtirpc/src/rtime.c:70:22: error: argument 'addrp' doesn't match prototype
In file included from e:/reactos/dll/3rdparty/libtirpc/tirpc/rpc/rpc.h:60:0,
                 from e:/reactos/dll/3rdparty/libtirpc/precomp.h:21:
e:/reactos/dll/3rdparty/libtirpc/tirpc/rpc/auth_des.h:124:12: error: prototype declaration
e:/reactos/dll/3rdparty/libtirpc/src/rtime.c:71:18: error: argument 'timep' doesn't match prototype
In file included from e:/reactos/dll/3rdparty/libtirpc/tirpc/rpc/rpc.h:60:0,
                 from e:/reactos/dll/3rdparty/libtirpc/precomp.h:21:
e:/reactos/dll/3rdparty/libtirpc/tirpc/rpc/auth_des.h:124:12: error: prototype declaration
e:/reactos/dll/3rdparty/libtirpc/src/rtime.c:72:18: error: argument 'timeout' doesn't match prototype
In file included from e:/reactos/dll/3rdparty/libtirpc/tirpc/rpc/rpc.h:60:0,
                 from e:/reactos/dll/3rdparty/libtirpc/precomp.h:21:
e:/reactos/dll/3rdparty/libtirpc/tirpc/rpc/auth_des.h:124:12: error: prototype declaration
e:/reactos/dll/3rdparty/libtirpc/src/rtime.c:73:1: error: number of arguments doesn't match prototype
In file included from e:/reactos/dll/3rdparty/libtirpc/tirpc/rpc/rpc.h:60:0,
                 from e:/reactos/dll/3rdparty/libtirpc/precomp.h:21:
e:/reactos/dll/3rdparty/libtirpc/tirpc/rpc/auth_des.h:124:12: error: prototype declaration
```

Reply:
Does 1feb8e6 helps with the issue?

Reply:
Yeap. Thanks!